lib/models/ObjectMD.ts

```diff
@@ -58,6 +58,7 @@ export type ObjectMDData = {
     'x-amz-server-side-encryption-customer-algorithm': string;
     'x-amz-website-redirect-location': string;
     'x-amz-scal-transition-in-progress'?: boolean;
+    'x-amz-scal-transition-time'?: string;
```

Comment:
(random line)
Should we update the `md-model-version` here https://github.com/scality/Arsenal/blob/4b642cf8b4de5a2d911a658884e3c2effbe79443/lib/constants.ts#L44 ?

Reply:
that is a good point, but i actually don't know:
* the field itself is "kind of" optional, so not really needed
* it seems many such "optional" fields have been added without bumping the model version
* but it may be better indeed to allow tracking more precisely (though it could backfire, if we end having huge switch case to handle all versions)...

Reply:
To me it looks like it allows to handle incompatible changes in the metadata, to allow handling both older and newer formats correctly by checking the version first.

My opinion is that if the change can be handled gracefully and robustly without having to bump the version number (like adding an optional new field), then it's not necessary. Some other devs can have a different opinion on this.